chains/ethereum/writer_methods.go

```diff
@@ -17,11 +17,11 @@ import (
 )
 
 // Number of blocks to wait for an finalization event
-var ExecuteBlockWatchLimit = 50
+var ExecuteBlockWatchLimit = 200
```

Comment:
In the PR addressing cancelling proposals we default to a 100 block limit, should we standardize that here as well?